apps/api-gateway/src/credential-definition/credential-definition.controller.ts

```diff
@@ -76,45 +77,48 @@ export class CredentialDefinitionController {
     summary: 'Fetch all credential definitions of provided organization id with pagination',
     description: 'Fetch all credential definitions from metadata saved in database of provided organization id.'
   })
-  @ApiQuery(
-    { name: 'pageNumber', required: false }
-  )
-  @ApiQuery(
-    { name: 'searchByText', required: false }
-  )
-  @ApiQuery(
-    { name: 'pageSize', required: false }
-  )
-  @ApiQuery(
-    { name: 'sorting', required: false }
-  )
-  @ApiQuery(
-    { name: 'sortByValue', required: false }
-  )
-  @ApiQuery(
-    { name: 'revocable', required: false }
-  )
   @Roles(OrgRoles.OWNER, OrgRoles.ADMIN, OrgRoles.ISSUER, OrgRoles.VERIFIER, OrgRoles.MEMBER)
   @UseGuards(AuthGuard('jwt'), OrgRolesGuard)
   async getAllCredDefs(
-    @Param('orgId') orgId: string,
+    @Param('orgId', new ParseUUIDPipe({exceptionFactory: (): Error => { throw new BadRequestException(`Invalid format for orgId`); }})) orgId: string,
     @Query() getAllCredDefs: GetAllCredDefsDto,
     @User() user: IUserRequestInterface,
     @Res() res: Response
-  ): Promise<object> {
+  ): Promise<Response> {
     const credentialsDefinitionDetails = await this.credentialDefinitionService.getAllCredDefs(
       getAllCredDefs,
       user,
       orgId
     );
-    const credDefResponse: IResponseType = {
+    const credDefResponse: IResponse = {
       statusCode: HttpStatus.OK,
       message: ResponseMessages.credentialDefinition.success.fetch,
-      data: credentialsDefinitionDetails.response
+      data: credentialsDefinitionDetails
     };
     return res.status(HttpStatus.OK).json(credDefResponse);
   }
 
+  @Get('/orgs/:orgId/bulk/cred-defs')
+  @ApiOperation({
+    summary: 'Fetch all credential definition for bulk opeartion',
+    description: 'Fetch all credential definition from metadata saved in database for bulk opeartion.'
```

Comment:
I think this message is misleading, Please check the message content

Reply:
Resolved